src/main/java/christmas/model/constant/DateConstant.java

```diff
@@ -0,0 +1,44 @@
+package christmas.model.constant;
+
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+public enum DateConstant {
+    MIN_DATE(1),
+    MAX_DATE(31),
+    XMAS(25);
+
+    public static final Set<Integer> WEEKDAYS = new HashSet<>();
+    public static final Set<Integer> WEEKENDS = new HashSet<>();
+    public static final Set<Integer> SPECIAL_DAYS = new HashSet<>(Arrays.asList(
+            3, 10, 17, 24, 25, 31
+    ));
```

Comment:
```suggestion
    public static final Set<Integer> SPECIAL_DAYS = Set.of(3, 10, 17, 24, 25, 31);
```
`Set.of` 를 활용해서 더 코드를 간결하게 만들 수 있을 것 같아요.

Reply:
SPECIAL_DAYS가 public으로 열려있으면서 불변이 아니므로 (HashMap) 외부에서 값을 수정하는 해킹이 가능할 것으로 보입니다!


예시입니다
```java
DateConstant.SPECIAL_DAYS.add(-1);
```